dll/win32/shlwapi/autocomp.cpp

```diff
@@ -251,11 +251,37 @@ BOOL CAutoCompleteEnumString::DoFileSystem(LPCWSTR pszQuery)
     return FALSE; // Not exact match
 }
 
+static LONG
```

Comment:
This smells like something a CRegKey object could do.

Reply:
Used `CRegKey`. OK.